riva/proto/riva_asr.proto

```diff
@@ -141,6 +141,7 @@ message RecognitionConfig {
     // Note: We only recognize the first channel by default.
     // To perform independent recognition on each channel set
     // `enable_separate_recognition_per_channel` to 'true'.
+    // Note: Only single channel files are supported.
```

Comment:
Minor comment. Can we say only one channel input is supported? since input can be from mic also.

Reply:
done